platform/landing-page/src/test/java/org/codice/ddf/landing/TestLandingPage.java

```diff
@@ -62,6 +62,12 @@ public static void setupLandingPage() throws IOException {
                 firstDateLeadingZeroes);
         sorted = Arrays.asList(firstDateLeadingZeroes, secondDateNoLeadingZeroes, noDate);
         landingPage.setAnnouncements(unsorted);
+        List<String> links = Arrays.asList("example, http://www.example.com",
```

Comment:
Can you please add some negative tests that handle invalid formats? Something like ",,," , ",example", "example,"